src/main/java/com/ericsson/ei/frontend/WebController.java

```diff
@@ -16,75 +16,59 @@
 */
 package com.ericsson.ei.frontend;
 
-import com.ericsson.ei.frontend.model.BackEndInformation;
-import com.ericsson.ei.frontend.utils.BackEndInstancesUtils;
+import com.ericsson.ei.frontend.utils.WebControllerUtils;
 
 import javax.servlet.http.HttpServletRequest;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-import lombok.Setter;
-
 @Controller
 public class WebController {
 
-    @Value("${ei.frontendServiceHost}")
-    private String frontendServiceHost;
-
-    @Setter
-    @Value("${ei.frontendServicePort}")
-    private int frontendServicePort;
-
-    @Value("${ei.frontendContextPath}")
-    private String frontendContextPath;
-
-    @Value("${ei.eiffelDocumentationUrls}")
-    private String eiffelDocumentationUrls;
-
-    @Value("${ei.useSecureHttpFrontend}")
-    private boolean useSecureHttpFrontend;
-
     @Autowired
-    private BackEndInstancesUtils backEndInstancesUtils;
+    private WebControllerUtils frontEndUtils;
 
+    public static final Logger LOG = LoggerFactory.getLogger(WebController.class);
     @RequestMapping("/")
     public String greeting(Model model) {
-
-        model.addAttribute("frontendServiceUrl", getFrontendServiceUrl());
-        String eiffelDocumentationUrlLinks = String.format("%s", eiffelDocumentationUrls);
+    	LOG.error("FRONTEND: " + frontEndUtils.getFrontEndServiceUrl());
```

Comment:
Why error here?
I FrontendService url can be both the bridge and eiBackend insance without bridege,, so pick a better variable name, e.g. EiServiceUrl or something.

Reply:
EIFrontEnds back end? or EI backend front end? this is confusing, this URL points to this service, this service is named EiFrontEnd, this this service URL is EiFrontEndUrl. The url you mentioned is not the back end, although this is a back end to the web gui. and a bridge. With just EiServiceUrl we do not know if it points to EI, or EI front end.